cmd/cri-containerd/options/options.go

```diff
@@ -239,6 +239,8 @@ func DefaultConfig() Config {
 				Runtime:       "io.containerd.runtime.v1.linux",
 				RuntimeEngine: "",
 				RuntimeRoot:   "",
+				RootDir:       "/var/lib/containerd",
```

Comment:
Why `ContainerdRootDir` and `ContainerdEndpoint` underlying does not work?

Reply:
don't know